code/modules/power/supermatter/supermatter.dm

```diff
@@ -463,10 +464,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		if(T)
 			var/obj/singularity/energy_ball/E = new(T)
 			E.energy = 200 //Gets us about 9 balls
-			explosion(get_turf(T), explosion_power * max(gasmix_power_ratio, 0.205) * 0.5 , explosion_power * max(gasmix_power_ratio, 0.205) + 2, explosion_power * max(gasmix_power_ratio, 0.205) + 4 , explosion_power * max(gasmix_power_ratio, 0.205) + 6, 1, 1)
-			qdel(src)
-			return
-	if(prob(power/50)) //near the threshold we have around 100% that the crystal will create the event
+	else if(power > EVENT_POWER_PENALTY_THRESHOLD && prob(power/100))
```

Comment:
So at 5k it's a 50% chance? just double checking to make sure that's what you want.

Reply:
ahhh fuk me i changed it to lower the chance if low power but then i added the check and forgot about that, no i want almost 100% near it

Reply:
just did a quick webedit thanks